src/js/common/actions/index.js

```diff
@@ -0,0 +1,7 @@
+export const LOG_OUT = 'LOG_OUT';
```

Comment:
Sounds good.  Keep in mind I'm fairly new to React, I'm not sure how faux pas it is for a component to import the actions of another component :smile:.  I yield to your best judgement.